src/porepy/grids/mortar_grid.py

```diff
@@ -198,8 +148,7 @@ def compute_geometry(self) -> None:
         We assume that they are not aligned with x (1d) or x, y (2d).
         """
         # Update the actual side grids
-        for g in self.side_grids.values():
-            g.compute_geometry()
+        [g.compute_geometry() for g in self.side_grids.values()]
```

Comment:
I'm pretty sure flake will not approve of this. If so, it should be changed back

Reply:
this must have been reverted when merging the mortar grid class form a different branch. My flake does not complain though